ruleset/sca/amazon/cis_amazon_linux_2022.yml

```diff
@@ -0,0 +1,5084 @@
+    compliance:
+      - cis: ["3.4.2.3"]
+      - cis_csc_v8: ["4.4", "4.8"]
+      - cis_csc_v7: ["9.4"]
+      - cmmc_v2.0:
+          [
+            "AC.L1-3.1.20",
+            "CM.L2-3.4.7",
+            "CM.L2-3.4.8",
+            "SC.L1-3.13.1",
+            "SC.L2-3.13.6",
+          ]
+      - pci_dss_3.2.1: ["1.1.6", "1.2.1", "2.2.2", "2.2.5", "1.1.4", "1.3.1"]
+      - pci_dss_4.0: ["1.2.1", "1.4.1", "1.2.5", "2.2.4", "6.4.1"]
+      - nist_sp_800-53: ["CA-9", "CM-7"]
+      - soc_2: ["CC6.3", "CC6.6"]
+    condition: any
+    rules:
+      - "c:rpm -q iptables-services -> r:^package iptables-services is not installed$"
+
+  # 3.4.2.4 Ensure iptables are flushed with nftables.
+  - id: 31095
+    title: "Ensure iptables are flushed with nftables."
+    description: "nftables is a replacement for iptables, ip6tables, ebtables and arptables."
+    rationale: "It is possible to mix iptables and nftables. However, this increases complexity and also the chance to introduce errors. For simplicity flush out all iptables rules, and ensure it is not loaded."
+    remediation: "Run the following commands to flush iptables: For iptables: # iptables -F For ip6tables: # ip6tables -F"
+    compliance:
+      - cis: ["3.4.2.4"]
+      - cis_csc_v8: ["4.4"]
+      - cis_csc_v7: ["9.4"]
+      - cmmc_v2.0:
+          ["AC.L1-3.1.20", "CM.L2-3.4.7", "SC.L1-3.13.1", "SC.L2-3.13.6"]
+      - pci_dss_3.2.1: ["1.1.4", "1.3.1"]
+      - pci_dss_4.0: ["1.2.1", "1.4.1"]
+      - nist_sp_800-53: ["CA-9"]
+      - soc_2: ["CC6.6"]
+    condition: all
+    rules:
+      - 'not c:sh -c "iptables -L | egrep -v  \"^target|^Chain\"" -> r:^\w+'
+      - 'not c:sh -c "ip6tables -L | egrep -v  \"^target|^Chain\"" -> r:^\w+'
+      - "c:rpm -q nftables -> r:^nftables-"
+
+  # 3.4.2.5 Ensure an nftables table exists.
+  - id: 31096
+    title: "Ensure an nftables table exists."
+    description: "Tables hold chains. Each table only has one address family and only applies to packets of this family. Tables can have one of five families."
+    rationale: "nftables doesn't have any default tables. Without a table being build, nftables will not filter network traffic."
+    impact: "Adding rules to a running nftables can cause loss of connectivity to the system."
+    remediation: "Run the following command to create a table in nftables # nft create table inet <table name> Example: # nft create table inet filter"
+    compliance:
+      - cis: ["3.4.2.5"]
+      - cis_csc_v8: ["4.4"]
+      - cis_csc_v7: ["9.4"]
+      - cmmc_v2.0:
+          ["AC.L1-3.1.20", "CM.L2-3.4.7", "SC.L1-3.13.1", "SC.L2-3.13.6"]
+      - pci_dss_3.2.1: ["1.1.4", "1.3.1"]
+      - pci_dss_4.0: ["1.2.1", "1.4.1"]
+      - nist_sp_800-53: ["CA-9"]
+      - soc_2: ["CC6.6"]
+    condition: all
+    rules:
+      - "c:nft list tables -> r:^table"
+      - "c:rpm -q nftables -> r:^nftables-"
+
+  # 3.4.2.6 Ensure nftables base chains exist.
+  # 3.4.2.9 Ensure nftables default deny firewall policy.
```

Comment:
Let's split controls. Each check should be unique. We did that in previous SCA files, but we decided to have each control unique although it is precisely the same check.

Reply:
Solved